static/components/PoiCard.js

```diff
@@ -6,7 +6,7 @@ class PoiCard extends React.Component {
 
   render() {
     return (
-      <div className='card'>
+      <div id={'card_'+this.props.details._id} className='card'>
         <PoiDistance
           lat1={this.props.details.position.latitude}
           lng1={this.props.details.position.longitude}
```

Comment:
instead of `lat1` `lng1` `lat2` `lng2`, can't you say something like below (needs to be adapted)?
```
<PoiDistance
    from={[this.props.details.position.latitude,this.props.details.position.longitude]}
    to={[this.props.userLat, this.props.userLng]} />
```